ui/app/AppLayouts/Chat/components/ProfilePopup.qml

```diff
@@ -396,9 +396,9 @@ ModalPopup {
             anchors.bottom: parent.bottom
             onClicked: {
                 if (isBlocked) {
-                    unblockContactConfirmationDialog.contactName = userName;
-                    unblockContactConfirmationDialog.contactAddress = fromAuthor;
-                    unblockContactConfirmationDialog.open();
+                  //  unblockContactConfirmationDialog.contactName = userName;
```

Comment:
Uncomment this once the missing UnblockContactConfirmationDialog component is added